paf-mvp-benchmark/OpenTelemetry/README.md

```diff
@@ -0,0 +1,78 @@
+# Tracing:
+
+## Links:
+https://opentelemetry.io/docs/instrumentation/js/getting-started/nodejs/
+
+## Install npm packages:
+### OpenTelemetry 
+`npm install @opentelemetry/sdk-node @opentelemetry/sdk-node`\
```

Comment:
Hint: use triple backquotes for blocks of code.

Reply:
Thanks, done.